x-pack/test/send_search_to_background_integration/tests/apps/management/search_sessions/sessions_management.ts

```diff
@@ -73,7 +77,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
 
       // NOTE: this test depends on the previous one passing
-      it('Reloads as new session from management', async () => {
+      it.skip('Reloads as new session from management', async () => {
```

Comment:
I'm not sure how this test is supposed to pass. I think the UI changed while the suite was skipped, and the test is obsolete now. 

cc @Dosant

Reply:
@tsullivan we removed the "reload" button in https://github.com/elastic/kibana/pull/90015 so this test case could be removed